addons/stock/static/src/js/basic_model.js

```diff
@@ -0,0 +1,15 @@
+odoo.define('stock.BasicModel', function (require) {
+"use strict";
+
+var BasicModel = require('web.BasicModel');
+
+BasicModel.include({
+
+    _invalidateCache: function (dataPoint) {
+        this._super.apply(this, arguments);
+        if (dataPoint.model === 'stock.warehouse') {
+            this.do_action('reload_context');
+        }
```

Comment:
is this how it's done for the companies ?

Reply:
Yes, when I checked todau, they add `&& !localStorage.getItem('running_tour')` to the condition to avoid reload during js tours. I will make this change to avoid same issue.

Reply:
you didn't add it it seems